examples/MvcWeb/TheraLang/Configuration/ResourceConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using MvcWeb.TheraLang.Entities;
+
+namespace MvcWeb.TheraLang.Configuration
+{
+    public class ResourceConfiguration : BaseEntityConfiguration<Resource>
+    {
+        public override void Configure(EntityTypeBuilder<Resource> builder)
+        {
+            builder.ToTable("Resources");
+            // builder.Property(b => b.CreatedDateUTC).HasDefaultValueSql("CURRENT_TIMESTAMP");
+
+            builder.HasKey(i => i.Id);
+            builder.Property(i => i.Id).ValueGeneratedOnAdd();
+            builder.Property(i => i.CreatedbyId).IsRequired();
+            builder.Property(i => i.CreatedDateUTC).IsRequired();
```

Comment:
Hey! that's great that you added BaseEntityConfiguration. It what I wanted 👍 . But that mean you do not need to configure these fields again in each entity, right?